packages/fxa-settings/src/pages/AccountRecoveryConfirmKey/en.ftl

```diff
@@ -0,0 +1,22 @@
+## AccountRecoveryConfirmKey page
+
+# Strings within the <span> elements appear as a subheading.
+# If more appropriate in a locale, the string within the <span>, "to continue to { $serviceName }" can stand alone as "Continue to { $serviceName }"
+account-recovery-confirm-key-heading-w-default-service = Reset password with account recovery key <span>to continue to account settings</span>
+# Strings within the <span> elements appear as a subheading.
+# If more appropriate in a locale, the string within the <span>, "to continue to { $serviceName }" can stand alone as "Continue to { $serviceName }"
+# { $serviceName } represents a product name (e.g., Mozilla VPN) that will be passed in as a variable
+account-recovery-confirm-key-heading-w-custom-service = Reset password with account recovery key <span>to continue to { $serviceName }</span>
+
+account-recovery-confirm-key-instructions = Please enter the one time use account recovery key you stored in a safe place to regain access to your Firefox Account.
+
+account-recovery-confirm-key-warning-message = <span>Note:</span> If you reset your password and don't have account recovery key saved, some of your data will be erased (including synced server data like history and bookmarks).
+# Prompts the user to enter their account recovery code
+account-recovery-confirm-key-input =
+  .label = Enter account recovery key
+# Clicking this button checks if the recovery key provided by the user is correct and associated with their account
+account-recovery-confirm-key-button = Confirm account recovery key
+# Error displayed in an alert banner when the recovery key confirmation is unsuccessful
+account-recovery-confirm-key-error-general = Recovery code is not valid
```

Comment:
Should this be `Account recovery key` instead of `Recovery code`?

Reply:
Yes, you're absolutely right!

@millsoper The original error message was `invalid hex string: null` which seems seems as a user-facing error message (in alert bar) - do you think `Account recovery key is not valid` is a reasonable alternative? Maybe this need product review?

Reply:
That's a great question -- I think it seems like a valid alternative but I would love a product review. I'll ping Vesta!

Reply:
I approved the PR as this string doesn't necessarily block localization (but would still hold off merging until you hear back from Vesta).

Reply:
@millsoper since this route won't be active for a while yet (and will be reviewed again when GQL is stitched in), do you think we can tentatively keep this updated error message with the comment in `index.tsx`?